tests/integration/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2019 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+cmake_minimum_required(VERSION 3.5)
+
+SET(OLP_CPP_SDK_INTEGRATION_TESTS_SOURCES
+    olp-cpp-sdk-core/OlpClientDefaultAsyncHttp.cpp
+)
+
+add_executable(olp-cpp-sdk-integration-tests
+    ${OLP_CPP_SDK_INTEGRATION_TESTS_SOURCES}
+)
+
+target_link_libraries(olp-cpp-sdk-integration-tests
+PRIVATE
+    gtest
```

Comment:
gtest_main already includes gtest. You only need gtest_main.
You are not linking twice to the same lib.

But i am pretty sure that online tests are not integration tests. Integration tests are e.g. CatalogClientTest with mocked Network. Online tests are functional tests. So probably you also need gmock lib